client/src/client_proxy.rs

```diff
@@ -47,7 +47,7 @@ use types::{
 
 const CLIENT_WALLET_MNEMONIC_FILE: &str = "client.mnemonic";
 const GAS_UNIT_PRICE: u64 = 0;
-const MAX_GAS_AMOUNT: u64 = 100_000;
+const MAX_GAS_AMOUNT: u64 = 140_000;
```

Comment:
Yea sure. This is the gas amount that we specify for the four basic transactions. This diff makes simple transaction a lot more expensive than they used to be because we are concatenating the event message ourselves, which caused the max amount to go up. I was thinking of whether we should define this constant somewhere in types as it was spread across different parts of the system for now. Any suggestions?